### PR TITLE
docs: refine KCLVM readme

### PR DESCRIPTION
**Pull request by Peefy**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [x] Y 

re #165. Refine KCL readme badges and documents.

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [x] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [x] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by ldxdl** [comment]
LGTM

**Review by ldxdl — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,45 +1,58 @@
-# KCL
+<h1 align="center">KCLVM</h1>
 
-![license](https://img.shields.io/badge/license-Apache--2.0-green.svg)
-[![Continuous Integration](https://github.com/KusionStack/KCLVM/actions/workflows/github-actions.yaml/badge.svg)](https://github.com/KusionStack/KCLVM/actions?query=branch%3Amain)
+<p align="center">
```
Comment (chai2010) [inline README.md:3]: add title at the first, for example: "The KCL Language"

File: README.md [hunk 2]
```diff
@@ -1,45 +1,58 @@
...
+  <img src="https://coveralls.io/repos/github/KusionStack/KCLVM/badge.svg">
+  <img src="https://img.shields.io/github/license/KusionStack/KCLVM.svg">
+  <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square">
+</p>
 
-Kusion Configuration Language (KCL) is an open source configuration language mainly used in [Kusion Stack](https://kusionstack.io). KCL is a statically typed language for configuration and policy scenarios, based on concepts such as declarative and Object-Oriented Programming (OOP) paradigms.
+<p align="center">
+	<img src="https://github.com/KusionStack/KCLVM/actions/workflows/ubuntu_test.yaml/badge.svg">
+  <img src="https://github.com/KusionStack/KCLVM/actions/workflows/macos_test.yaml/badge.svg">
+  <img src="https://github.com/KusionStack/KCLVM/actions/workflows/windows_test.yaml/badge.svg">
```
Comment (chai2010) [inline README.md:21]: simplify action name, for example: rename "build-and-test-macos" to "macos"

File: README.md [hunk 3]
```diff
@@ -1,45 +1,58 @@
...
+<h1 align="center">KCLVM</h1>
 
-![license](https://img.shields.io/badge/license-Apache--2.0-green.svg)
-[![Continuous Integration](https://github.com/KusionStack/KCLVM/actions/workflows/github-actions.yaml/badge.svg)](https://github.com/KusionStack/KCLVM/actions?query=branch%3Amain)
+<p align="center">
+<a href="./README.md">English</a> | <a href="./README-zh.md">简体中文</a>
+</p>
+<p align="center">
+<a href="#introduction">Introduction</a> | <a href="#features">Features</a> | <a href="#features">Features</a> | <a href="#installation">Installation</a> | <a href="#what-is-it-for">What is it for</a> | <a href="#documentation">Documentation</a> | <a href="#contributing">Contributing</a> | <a href="#roadmap">Roadmap</a> | <a href="#license">License</a>
+</p>
```
Comment (chai2010) [inline README.md:8]: menu should use short name

Comment (chai2010) [reply README.md:8]: dup: Features

File: README.md (hunk 2 above)
Comment (chai2010) [reply README.md:21]: only include one named "KCL" action badge, ant move to the first.

File: README.md (hunk 1 above)
Comment (Peefy) [reply README.md:3]: > add title at the first, for example: "The KCL Language"

Add a title.

File: README.md (hunk 2 above)
Comment (Peefy) [reply README.md]: > only include one named "KCL" action badge, ant move to the first.

fixed.

File: README.md (hunk 3 above)
Comment (Peefy) [reply README.md:8]: > dup: Features

fixed.

Comment (Peefy) [reply README.md:8]: > menu should use short name

fixed.

Comment (chai2010) [reply README.md:8]: add “KCL” build statue badge at head

**Review by chai2010 — APPROVED** [review]
LGTM

**Review by He1pa — APPROVED** [review]
lgtm

**Review by zong-zhe — APPROVED** [review]
LGTM

**Review by He1pa — APPROVED** [review]
lgtm

**Review by zong-zhe — APPROVED** [review]
LGTM

File: README.md (hunk 3 above)
Comment (Peefy) [reply README.md:8]: > add “KCL” build statue badge at head

fixed.

**Review by ldxdl — DISMISSED** [review]

**Review by zong-zhe — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -1,52 +1,61 @@
...
-  + Code organization: modules and packages.
-  + [Plug-in system](https://github.com/KusionStack/kcl-plugin): reuse common programming language ecology.
-  + [OpenAPI model support](https://github.com/KusionStack/kcl-openapi): Swagger and KCL schema bidirectional conversion, Kubernetes CRD conversion to KCL schema.
-+ **High Performance**
-  + Works with the LLVM optimizer, supports compilation to native code and formats like WASM and executes efficiently.
+## Features
 
-## Installing & Documentation
++ **Easy to use**: Originated from high-level languages ​​such as Python and Golang, incorporating functional language features, low side effects.
++ **Well-designed**: Independent spec-driven deigned syntax, semantics, runtime and system modules.
```
Comment (ldxdl) [inline README.md:25]: Independent Spec-driven syntax, semantics, runtime and system modules design.

Comment (Peefy) [reply README.md]: > Independent Spec-driven syntax, semantics, runtime and system modules design.

fixed.

**Review by zong-zhe — APPROVED** [review]

**Review by He1pa — APPROVED** [review]
lgtm

**Pull request merged**